app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java

```diff
@@ -99,6 +99,8 @@ public void onBindViewHolder(@NonNull PictureViewHolder holder, int position) {
             holder.year.setText(artwork.getYear());
             holder.description.setText(shortenDescription(artwork.getSummary()));
             holder.score.setText("+" + artwork.getScore() + " pts");
+            holder.location.setText(artwork.getCity());
+
```

Comment:
are we sure we get the city for each artwork that is scanned? Or what happens if we don't have a city associated with the artwork?